frontend/src/components/FilterBar/FilterBar.tsx

```diff
@@ -11,7 +12,14 @@ export let FilterBar = withStore('method', 'allowDrivingTime')(({ store }) => {
       <span>Method</span>
       <DropDownMenu
         className='DropDownMenu -Compact'
-        onChange={(_e, _i, value) => store.set('method')(value)}
+        onChange={(_e, _i, value) => {
+          ReactGA.event({
+            category: 'Adequacy',
+            action: 'Selected an adequacy type',
```

Comment:
Should we use present tense?

Reply:
It's better practice for these to be past tense, but you did remind me to go through and check - I just updated one from present tense to past tense so that they are all consistent.